scripts/gen_site.sh

```diff
@@ -44,6 +44,10 @@ babel --source-maps --minified --no-comments --presets minify \
   tmp/js/events.js \
   --out-file generated/js/all.min.js
 
+babel --source-maps --minified --no-comments --presets minify \
```

Comment:
Why add this as a separate file instead of part of line starting at 25?

Reply:
The only reason was to load it when necessary instead of inside the large `all.min.js` but I believe the homepage is going to be the first page users go to anyway?

Reply:
Yep, Let's keep it separate.